File: core/feature04b_test.go
package main

import (
	"testing"
)

var (
	feature04BNames      = []string{"A", "B"}
	feature04BAccuracies = []string{"%.0f", "%.0f"}
	feature04BTypes      = []bool{true, true}
)

func Test01Feature04B(t *testing.T) {
	arguments := map[string]interface{}{"M": 773}
	expected := map[string]interface{}{"A": 40, "B": 50}
	genericTester(t, feature04B(arguments), expected, feature04BNames, feature04BAccuracies, feature04BTypes)
}

File: core/feature04b_test.js
var feature04BNames = ["A", "B"];
var feature04BAccuracies = [0, 0];
var feature04BTypes = [true, true];

function test01Feature04B() {
  var arguments = {"M": 773};
  var expected = {"A": 40, "B": 50};
  genericTester(feature04b_core(arguments), expected, feature04BNames, feature04BAccuracies, feature04BTypes);
}

test01Feature04B();
